presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1606,6 +1606,7 @@ private Scope computeAndAssignOrderByScopeWithAggregation(OrderBy node, Scope ou
                     .collect(toImmutableList());
 
             Scope orderByAggregationScope = Scope.builder()
+                    .withRelationId(RelationId.anonymous())
```

Comment:
`orderByAggregationScope` is an artificial concept required only so that it's possible to do `aggregation expression -> symbol` mapping during planning (`TranslationMap`). The whole `computeAndAssignOrderByScopeWithAggregation` will be gone when https://github.com/prestodb/presto/issues/7398 is implemented. No need to do further hacks around this method